README.md

```diff
@@ -0,0 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# portfolio project
+
+> this is my first portifolio.
+
+![screenshot](./images/screenshot/screenshot-mobile-version.png)
+
+home page for mobile skeleton
+
+## Built With
+
+- HTML 
+- CSS
+- No framwork used in this project
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**to start you need t create a repository and clone it to your local computer**
```

Comment:
- [ ] Please correct your typo error here.

- [ ] You are not using the proper commit message rules. Going forward, please ensure that your commit messages are **imperative** and **capitalized**. They should also be short, if possible. For example:
  - **adding files to the mobile portfolio** should have been **Add files to the mobile portfolio**
  - **adding correction from linters error to styles.css file , image, Readme correction of image logation** should have been **Add correction from linters error to styles.css file, image, Readme correction of image location**